src/app/public/modules/image/index.ts

```diff
@@ -0,0 +1,7 @@
+export {
```

Comment:
Mind if we remove all `index.ts` files except for `./index.ts` and `./src/app/public/index.ts`? I need to update the style guide, but we're going to move away from using exported "barrel" files since they can easily cause circular import references.